src/app/shared/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -50,6 +51,7 @@ export default function Breadcrumbs(props: BreadcrumbsProps): JSX.Element {
               item={items[i]}
               isHiddenInList
               totalBreadcrumbsLength={items.length}
+              items={items}
```

Comment:
Passing `JSX.Element[]` as list items is causing some click/hover problems (and is not a good practice).
On the video you can see that if the user don't click exactly on the text it looks like its not working.

https://user-images.githubusercontent.com/34165099/214921232-9bd1b252-385e-4a95-adca-d43719a88598.mov

Reply:
done